sources/modules/VEScenario/R/RunScenarios.R

```diff
@@ -506,8 +496,7 @@ RunScenarios <- function(L){
                        return(output)
                      },
                      label = ScenarioName_,
-                     globals = TRUE,
-                     seed = TRUE),
```

Comment:
@arashasadabadi, your changes here omitted `seed = TRUE`, which is why you still see the RNG warning. I'll make a commit to your PR to put this back in. See https://github.com/VisionEval/VisionEval-Dev/commit/d4f0809f31d7529abec46d1de42bd18e2e9dffde.

Reply:
Fixed in 4442f1b